docs/plans/index.html

```
@@ -0,0 +1,461 @@
+    }
+  </style>
+</head>
+<body>
+
+  <div class="gallery-header">
+    <h1>Plans Library</h1>
+    <p>1 plans &mdash; click any card to open it</p>
+  </div>
+
+  <div class="toolbar">
+    <input type="text" class="search-box" id="searchBox"
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)